browser/misc_metrics/theme_metrics.cc

```diff
@@ -16,10 +17,14 @@ namespace misc_metrics {
 
 ThemeMetrics::ThemeMetrics(ThemeService* theme_service)
     : theme_service_(theme_service) {
-  pref_change_registrar_.Init(g_browser_process->local_state());
-  pref_change_registrar_.Add(kBraveDarkMode,
-                             base::BindRepeating(&ThemeMetrics::ReportMetrics,
-                                                 base::Unretained(this)));
+  if (g_browser_process->local_state()) {
+    pref_change_registrar_.Init(g_browser_process->local_state());
+    pref_change_registrar_.Add(kBraveDarkMode,
```

Comment:
why is `kBraveDarkMode` in local state?

Reply:
https://github.com/brave/brave-core/pull/4210 cc: @simonhong

Reply:
I think this needs a follow-up issue to move it to profile prefs

Reply:
At that time, upstream doesn't support per-profile theme mode.
As upstream started to have theme mode also, we don't need to have our own code for it.
Filed issue for it - https://github.com/brave/brave-browser/issues/40863